project/src/main/webapp/create-event.html

```diff
@@ -144,22 +124,43 @@ <h3>Create an Event</h3>
         />
       </div>
       <br />
-      <div class="input-group">
-        <div class="input-group-prepend">
-          <span class="input-group-text" id="basic-addon1">Wanted</span>
+      <div class="row">
+        <div class="input-group col-md-4">
+          <div class="input-group-prepend">
+            <span class="input-group-text" id="basic-addon1">Wanted</span>
+          </div>
+          <input
+            type="text"
+            class="form-control"
+            placeholder="Volunteer Type"
+            aria-describedby="basic-addon1"
+          />
+        </div>
+        <div class="input-group col-md-1">
+          <input
+            type="text"
+            class="form-control"
+            placeholder="Quantity Needed"
+            aria-describedby="basic-addon1"
+          />
+        </div>
+        <div class="input-group col-md-4" id="skills-div">
+          <div class="input-group-prepend">
+            <label class="input-group-text" for="skills">Skills</label>
+          </div>
+          <select
+            id="skills"
+            name="skills"
+            data-role="tagsinput"
+            multiple
+          ></select>
+          <button 
+            type="button"
+            class="btn btn-light prefilled-pick"
+            onclick="togglePrefilledSkills();"
+          >+ Pick</button>
+          <div id="prefilled-skills" class="overflow-auto prefilled-pool"></div>
         </div>
```

Comment:
Skills are on the volunteering form ~ move this to volunteering opportunity.html.